module/VuFind/src/VuFind/Db/Entity/SearchEntityInterface.php

```diff
@@ -40,4 +42,170 @@
  */
 interface SearchEntityInterface extends EntityInterface
 {
+    /**
+     * Get identifier (returns null for an uninitialized or non-persisted object).
+     *
+     * @return ?int
+     */
+    public function getId(): ?int;
+
+    /**
+     * Get user.
+     *
+     * @return int
+     */
+    public function getUserId(): int;
+
+    /**
+     * Set user.
+     *
+     * @param int $user User
+     *
+     * @return SearchEntityInterface
+     */
+    public function setUserId(int $user): SearchEntityInterface;
+
+    /**
+     * Get session.
```

Comment:
I'd add "identifier" to the comment here and in the setter -- i.e. "Get session identifier" -- just to be totally clear.